openpathsampling/tests/test_data/ops1_api.txt

```diff
@@ -0,0 +1,270 @@
+openpathsampling.OptionalEnsemble
+openpathsampling.PartInBFixedLengthTPSNetwork
+openpathsampling.PartInBFixedLengthTPSTransition
+openpathsampling.PartInXEnsemble
+openpathsampling.PartOutXEnsemble
+openpathsampling.PartialAcceptanceSequentialMoveChange
+openpathsampling.PartialAcceptanceSequentialMover
+openpathsampling.PathDensityHistogram
+openpathsampling.PathMover
+openpathsampling.PathReversalMover
+openpathsampling.PathReversalSet
+openpathsampling.PathSampling
+openpathsampling.PathSimulator
+openpathsampling.PathSimulatorMoveChange
+openpathsampling.PathSimulatorMover
+openpathsampling.PeriodicCVDefinedVolume
+openpathsampling.PeriodicVolumeInterfaceSet
+openpathsampling.PrefixTrajectoryEnsemble
+openpathsampling.PyEMMAFeaturizerCV
+openpathsampling.RandomAllowedChoiceMover
+openpathsampling.RandomChoiceMoveChange
+openpathsampling.RandomChoiceMover
+openpathsampling.RandomSubtrajectorySelectMover
+openpathsampling.RandomVelocities
+openpathsampling.RejectedMaxLengthSampleMoveChange
+openpathsampling.RejectedNaNSampleMoveChange
+openpathsampling.RejectedSampleMoveChange
+openpathsampling.RelativeComplementVolume
+openpathsampling.ReplicaExchangeMover
+openpathsampling.ReplicaNetwork
+openpathsampling.ReplicaNetworkGraph
+openpathsampling.ReversedTrajectoryEnsemble
+openpathsampling.SRTISBiasFromNetwork
+openpathsampling.SRTISScheme
+openpathsampling.Sample
+openpathsampling.SampleDetails
+openpathsampling.SampleMoveChange
+openpathsampling.SampleMover
+openpathsampling.SampleSet
+openpathsampling.SelectionMover
+openpathsampling.SequentialEnsemble
+openpathsampling.SequentialMoveChange
+openpathsampling.SequentialMover
+openpathsampling.ShootFromSnapshotsSimulation
+openpathsampling.ShootingPointAnalysis
+openpathsampling.ShootingPointSelector
+openpathsampling.SingleAtomVelocityDirectionModifier
+openpathsampling.SingleFrameEnsemble
+openpathsampling.SingleReplicaMinusMover
+openpathsampling.SnapshotByCoordinateDict
+openpathsampling.StateSwapMover
+openpathsampling.StepVisualizer2D
+openpathsampling.Storage
+openpathsampling.SubMoveChange
+openpathsampling.SubPathMover
+openpathsampling.SubtrajectorySelectMover
+openpathsampling.SuffixTrajectoryEnsemble
+openpathsampling.SymmetricDifferenceVolume
+openpathsampling.TISEnsemble
+openpathsampling.TISTransition
+openpathsampling.TPSNetwork
+openpathsampling.TPSTransition
+openpathsampling.engines.Trajectory
+openpathsampling.TrajectorySegmentContainer
+openpathsampling.TrajectoryTransitionAnalysis
+openpathsampling.Transition
+openpathsampling.TransitionNetwork
+openpathsampling.TwoWayShootingMover
+openpathsampling.UniformSelector
+openpathsampling.UnionEnsemble
+openpathsampling.UnionVolume
+openpathsampling.VelocityDirectionModifier
+openpathsampling.Volume
+openpathsampling.VolumeCombination
+openpathsampling.VolumeEnsemble
+openpathsampling.VolumeInterfaceSet
+openpathsampling.WrappedEnsemble
+openpathsampling.condense_repeats
+openpathsampling.git_HEAD
```

Comment:
What would this API be?

Reply:
`git_HEAD` is trick that I added to get around our static values in the `version.py` file. If you have a developer installation of OPS, the actual git revision is available from `paths.git_HEAD()`. (Basically, it gives the output of `git -C $OPS_DIR rev-parse HEAD`.)

In the future, I'll add the dynamic `version.py` that I've used in more recent projects, but not before 1.0.